source/release-notes-pxc-v8.0.25.rst

```diff
@@ -0,0 +1,55 @@
+.. _pdpxc.release_notes_8.0.25:
+
+********************************************************************************
+|pdmysql| 8.0.25 using |PXC|
+********************************************************************************
+
+:Date: 
+:Installation: :ref:`installation`
+
+|pdmysql| is a single solution with the best and most critical enterprise components from the |mysql| open source community, designed and tested to work together. 
+
+|pdmysql| provides two deployment variants: one is based on |PS| and another one is based on |PXC|. 
+
+This release of |pdmysql| is focused on the |PXC|-based deployment variant. It is based on `Percona XtraDB Cluster 8.0.25-15.1 <https://www.percona.com/doc/percona-xtradb-cluster/8.0/release-notes/Percona-XtraDB-Cluster-8.0.25-15.1.html>`_ 
+
+Release Highlights
+==================
+
+`Online Schema Upgrade <https://www.percona.com/doc/percona-xtradb-cluster/8.0/features/online-schema-upgrade.html>`_ is introduced in |PXC|.
```

Comment:
As this is a key feature highlight, could we add at least one or more two sentences about it? Why should users care about this?

Reply:
The new feature is Non-Blocking Operations mode. When performing an online schema upgrade in this mode, a DDL operation only acquires a metadata lock on the table or schema during the execution of updating the table or schema definitions.

Reply:
Thank you for the technical background - so what does this mean in practice for the user? What's the benefit? Will this make these operations to perform faster? How will it help in concurrent environments?

Reply:
This mode is similar to Total Order Isolation (TOI) mode but avoids the issue of long-running DDL statements blocking other updates in the cluster.

Reply:
Updated